src/components/Layout/components/getThemeValue.ts

```diff
@@ -0,0 +1,31 @@
+import {LayoutProps, LayoutTheme, MediaPartial} from '../types';
+
+/**
+ * Select provided prop value from LayoutTheme. Depends on the current screen media query
```

Comment:
this (and others) should be a simple comment, not jsdoc

Reply:
why?
all modern code editors support `jsdoc` in references.
You can easily write `md` syntax here like:
<img width="770" alt="image" src="https://user-images.githubusercontent.com/27916444/209137898-2088492f-a671-439c-83bb-9fe1305b8f18.png">

Reply:
ok, l'll fix this. 
But i think project don't need this rules (@return statement in jsdoc for example) cos we have typescript in the project and use jsdoc only for annotations